webpack/assets/javascripts/react_app/components/AuditsList/index.js

```diff
@@ -107,6 +120,21 @@ const AuditsList = ({ data: { audits } }) => (
             <ActionLinks allowedActions={allowedActions} />
           </Row>
 
+          <Row>
+            <Col sm={10} className="request_uuid_column">
+              <Row>
+                <Col md={2}>
+                  <span>{__('Request UUID')}</span>
+                </Col>
+                <Col md={10}>
+                  <strong>
```

Comment:
Could you use css to make it strong?
It's not a good practice to render big html dom trees inside `strong` or `b`.

```css
font-weight: bold;
```